pyproject.toml

```diff
@@ -12,14 +12,14 @@ authors = [
 readme = "README.md"
 requires-python = ">=3.10"
 dependencies = [
-  "huggingface-hub>=0.28.0",
```

Comment:
We recently required `"huggingface-hub>=0.28.0"` to support inference providers:
- #422

Reply:
Thanks for review. 
I fixed huggingface-hub dep.

https://github.com/huggingface/smolagents/pull/374/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711R15